homework_1/task_2/main.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"code-cadets-2021/homework_1/task_2/tax"
+)
+
+var taxBuckets = []tax.Bucket{
+	{
+		AppliedAbove: 1_000,
```

Comment:
Where did you find this convention? I haven't seen it before in Go and Goland reports an error, even though code does work.

Reply:
Pajtončina je prvi uveo to, kao za readability. Tu je gramatika za go: https://golang.org/ref/spec#Integer_literals

Reply:
English please.

> Mozda ti je konfiguriran na stariju verziju jezika?
Yep that could be the case, but I won't look into it. We don't usually write numbers like that (actually we almost never have numbers anywhere in code, as we define them in the application configuration, while in code we do various other stuff e.g. mapping models).

Reply:
Better to include the underscores right away than to later have to deal with the question:
> Nikola... Di su nule od stotice, a di od tisucice?

Reply:
> e.g. mapping models).

You're confusing "e.g." with "i.e."